fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/AbstractNonRdfSourcePersister.java

```diff
@@ -88,12 +88,13 @@ protected void persistNonRDFSource(final ResourceOperation operation,
         if (forExternalBinary(nonRdfSourceOperation)) {
             outcome = null;
         } else {
+            final var transmissionDigestAlg = objectSession.getObjectDigestAlgorithm();
             final var providedDigests = nonRdfSourceOperation.getContentDigests();
             // Wrap binary stream in digest computing wrapper, requesting
             final var multiDigestWrapper = new MultiDigestInputStreamWrapper(
                     nonRdfSourceOperation.getContentStream(),
                     providedDigests,
-                    Arrays.asList(objectSession.getObjectDigestAlgorithm()));
+                    Arrays.asList(transmissionDigestAlg));
```

Comment:
No need to change, but this can now be constructed using `List.of()`.

Reply:
Sure, I'll swap it